mlir/test/auto_e2e/lit.site.cfg.py.in

```diff
@@ -0,0 +1,57 @@
+@LIT_SITE_CFG_IN_HEADER@
+
+import sys
```

Comment:
@yiqian1 same here. it'd be great if a document describes what's under the hood between `check-mlir` and `check-e2e`.